scripts/get_openstack_plaintext_docs.sh

```diff
@@ -44,9 +44,7 @@ _OS_PROJECTS="nova horizon keystone neutron cinder manila glance swift ceilomete
 octavia designate heat placement ironic barbican aodh watcher adjutant blazar \
 cyborg magnum mistral skyline-apiserver skyline-console storlets \
 venus vitrage zun python-openstackclient tempest trove zaqar masakari"
-if [ -z "$OS_PROJECTS" ]; then
-    OS_PROJECTS=$_OS_PROJECTS
-fi
+OS_PROJECTS=${OS_PROJECTS:-$_OS_PROJECTS}
```

Comment:
I personally prefer leaving this code the same way as it was before because it will also work if the env var is defined as an empty string.

So we could change line 17 to `set -eo pipefail` to solve the problem, or create a slightly more complex condition:
```
if [ -n "${OS_PROJECTS+x}" ] && [ -n "$OS_PROJECTS" ]; then
```

Reply:
Ok, a tiny bit puzzled by what was the issue at the beginning since the `:` was there :see_no_evil: 

> In any case, the patch is ready to merge, and it's not worth wasting more time on this.

:+1: marking as resolved and merging, hope it is ok